.travis.yml

```diff
@@ -40,6 +40,11 @@ matrix:
   exclude:
     - os: osx
       compiler: gcc
+  include:
+    - if: branch = master
+      compiler: clang
+      install:
+      script: brew install --HEAD axel && brew test axel
```

Comment:
What does this exactly do? It tries to build axel through brew? How does that work? where's the recipe?